test/functional/tools/sam_to_bam_native.xml

```diff
@@ -0,0 +1,27 @@
+<tool id="sam_to_bam_native_conversion" name="Test sam to bam native conversion">
+    <requirements>
+        <requirement type="package" version="1.6">samtools</requirement>
+    </requirements>
+    <command><![CDATA[
+        samtools view
+            -b
+            -@ \${GALAXY_SLOTS:-2}
+            -o '${output}'
+            '$input'
+    ]]>
+    </command>
+    <inputs>
+        <param name="input" type="data" format="sam" label="SAM file"/>
```

Comment:
This still doesn't test the converter, it just runs this tool. You should be able to specify as input `bam_native` but provide a sam file in the test. Copy that to an output, and the output should be an unsorted bam file.

Reply:
Ah, I see. No this test is testing something different. It tests that Galaxy is not sorting this file.
Do you want to see an additional test for the implicit converter as well?